rustfmt.toml

```diff
@@ -1,2 +1,3 @@
 max_width = 90
 newline_style = "Unix"
+# format_code_in_doc_comments = true
```

Comment:
This is a nightly feature.

Nightly format could be run with `cargo +nightly fmt`.

But just leaving it here uncommented breaks regular `catgo fmt` calls.